config/database.yml

```
@@ -3,18 +3,19 @@
 
     config = ClowderCommonRuby::Config.load
 
+    db_service = config.database.hostname
     username = config.database.username
     password = config.database.password
     host = config.database.hostname
     database = config.database.name
+    port = config.database.port
   else
-
     db_service = ENV.fetch("DATABASE_SERVICE_NAME","").upcase.sub("-", "_")
-
     username = ENV.key?("POSTGRESQL_ADMIN_PASSWORD") ? "postgres" : ENV["POSTGRESQL_USER"]
     password = ENV.key?("POSTGRESQL_ADMIN_PASSWORD") ? ENV["POSTGRESQL_ADMIN_PASSWORD"] : ENV["POSTGRESQL_PASSWORD"]
     host = ENV.fetch("#{db_service}_SERVICE_HOST")
```

Let's just remove db_service:

```suggestion
    host = ENV.fetch("POSTGRES_SERVICE_HOST")
```